src/DependencyInjection/Configuration.php

```diff
@@ -55,6 +70,12 @@ public function getConfigTreeBuilder()
                         ->thenInvalid('%s')
                     ->end()
                 ->end()
+                ->booleanNode('image_cache')
```

Comment:
I don't like the naming of the variable. What about `rebuild_image_cache` or `recreate_images` or `bypass_image_cache`?